cmd/epp/main.go

```diff
@@ -58,8 +50,7 @@ var (
 		"grpcHealthPort",
 		9003,
 		"The port used for gRPC liveness and readiness probes")
-	metricsPort = flag.Int(
-		"metricsPort", 9090, "The metrics port")
+	metricsAddr                = flag.String("metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
Can you use `9090` so we don't need to modify all mentioning of default metrics port in the project: https://github.com/search?q=repo%3Akubernetes-sigs%2Fgateway-api-inference-extension%209090&type=code